packages/yoga/src/Drawer/web/Drawer.jsx

```diff
@@ -1,28 +1,35 @@
 import React from 'react';
 import styled from 'styled-components';
 import { node, number } from 'prop-types';
+import { theme } from '@gympass/yoga';
 import Dialog from '../../Dialog';
 
 const StyledDrawer = styled(Dialog)`
+  position: absolute;
+  right: 0;
+
+  align-self: flex-end;
+
+  height: 100%;
+  padding: ${theme.spacing.zero} !important;
+  border-radius: ${theme.borders.zero} !important;
```

Comment:
If you are using `!important` here probably something is not right with the `css` structure. 
Can you check?